dev/com.ibm.ws.logging_fat/publish/servers/TraceSourceHandlerServer/server.xml

```diff
@@ -4,10 +4,9 @@
 	<featureManager>
 		<feature>jsp-2.2</feature>
 		<feature>sampleTraceSourceHandler-1.0</feature>
-		<feature>jaxrs-1.1</feature>
```

Comment:
Is jaxrs simply not required or do we need to upgrade to a later level?

Reply:
It doesn't seem required at all. I also did try upgrade to later level but it has some feature conflicts with `sampleTraceSourceHandler-1.0`